src/common/__tests__/merjoonTransformer.test.ts

```diff
@@ -148,5 +168,63 @@ describe('MerjoonTransformer', () => {
         });
       });
     });
+
+    it('should break loop and return undefined if new value is undefined', () => {
```

Comment:
do not it and describe on the same level, let's wrap with describes to keep things constant